CONTRIBUTING.md

```diff
@@ -7,9 +7,15 @@ Please make sure the following company details are provided:
 * Name;
 * Website (linked from the company's name);
 * Careers website, if exists (linked from the :rocket:);
+* Github (or another git-repository manager) account, if exists and contains at least one relevant, and considered active, repository (linked from the :octocat:);
```

Comment:
Suggestion to improve readability: 

     Github account, or another git-repository manager account, if exists and contains at least one relevant and active repository (linked from the :octocat:);